src/ray/raylet/local_object_manager.cc

```diff
@@ -233,7 +268,9 @@ void LocalObjectManager::SpillObjectsInternal(
           request.add_object_ids_to_spill(object_id.Binary());
           auto it = objects_pending_spill_.find(object_id);
           RAY_CHECK(it != objects_pending_spill_.end());
-          request.add_owner_addresses()->MergeFrom(it->second.second);
+          // TODO (yic): To support ownership transfer
```

Comment:
I'll add the support in future PR. Right now, only take care of non-spilled objects.

Reply:
yeah... it's a bit issue.

Reply:
Can you actually comment what needs to be improved for object transfer?